### ISSUE TITLE
files not added under solution control

### ISSUE THREAD

**Issue by lucamorelli** [aspnet/Tooling#429]
Hi,
  this is a long time problem, and I don't know if has been fixed in  update 2

![untitled2](https://cloud.githubusercontent.com/assets/127777/13862340/e03fdf1c-ec92-11e5-9a57-65de5dec5ba6.png)

  Happens very often that if I copy and paste class files under solution explorer the new pasted files is not added un solution control, so if I don't go in the untracked files section and add manually to the commit, the file remains local with all the related problems.
  This is an inconsistent behavior, since adding manually a file is added in sc too.

**Comment by sayedihashimi** [comment]
@lucamorelli is this using git or tfs? If tfs is it a local or server workspace?

**Comment by lucamorelli** [comment]
Git

**Comment by sayedihashimi** [comment]
@lucamorelli here is what I just tried.

 1. Create new project
 1. Add to local git repo
 1. Add new class `sample.cs` in Models folder
 1. Select `sample.cs` in solution explorer and CTRL+C
 1. Paste into Controllers folder

That worked fine for me. I pasted the file into many folders and after each paste a new entry appeared in pending changes. Did I get the steps right here or is there some other way I can reproduce this?
